upgrader/web/index.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
+define('ROOT_PATH', realpath(__DIR__ . '/../..'));
```

Comment:
shouldn't it be 
defined('CONSTANT') or define('CONSTANT', 'SomeDefaultValue');

Reply:
I guess define('CONSTANT', 'SomeDefaultValue'); is correct.